client/packages/host/src/useInitPlugins.ts

```diff
@@ -40,7 +41,7 @@ export const useInitPlugins = () => {
   useEffect(() => {
     if (
       process.env['NODE_ENV'] === 'production' ||
-      Environment.LOAD_REMOTE_PLUGINS
+      featureFlags.loadRemotePluginsInDev
```

Comment:
I think this won't quite work featureFlags won't be available at this stage yet ?

I haven't really used this feature that much, it was useful when originally doing plugin work, not so much since. 

I think it's ok to remove this for now (the alternative is to build the front end and access it through 'server', which is ok.

Reply:
Ohh true, I didn't try that sorry. Ok, will remove for this case - good thing to be aware of if we need flags available on startup in future though!